Mathlib/CategoryTheory/Limits/ConcreteCategory.lean

```diff
@@ -53,16 +53,30 @@ theorem Concrete.to_product_injective_of_isLimit {D : Cone F} (hD : IsLimit D) :
   apply Subtype.ext
 #align category_theory.limits.concrete.to_product_injective_of_is_limit CategoryTheory.Limits.Concrete.to_product_injective_of_isLimit
 
-theorem Concrete.isLimit_ext {D : Cone F} (hD : IsLimit D) (x y : D.pt) :
+theorem isLimit_ext {D : Cone F} (hD : IsLimit D) (x y : D.pt) :
     (∀ j, D.π.app j x = D.π.app j y) → x = y := fun h =>
   Concrete.to_product_injective_of_isLimit _ hD (funext h)
 #align category_theory.limits.concrete.is_limit_ext CategoryTheory.Limits.Concrete.isLimit_ext
 
-theorem Concrete.limit_ext [HasLimit F] (x y : ↑(limit F)) :
+theorem limit_ext [HasLimit F] (x y : ↑(limit F)) :
     (∀ j, limit.π F j x = limit.π F j y) → x = y :=
   Concrete.isLimit_ext F (limit.isLimit _) _ _
 #align category_theory.limits.concrete.limit_ext CategoryTheory.Limits.Concrete.limit_ext
 
+section Surjective
+
+/--
+Given surjections `⋯ ⟶ Xₙ₊₁ ⟶ Xₙ ⟶ ⋯ ⟶ X₀` in a concrete category whose forgetful functor
+preserves sequential limits, the projection map `lim Xₙ ⟶ X₀` is surjective.
+-/
+lemma limit_of_surjections_surjective {C : Type u} [Category.{v} C] [ConcreteCategory.{v} C]
```

Comment:
I think the name given by the naming convention would be something like `surjective_π_app_zero_of_surjective_map`.